src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/StringCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret text")
 public class StringCredentials extends BaseStandardCredentials {
 
-    public Control secret = control("secret");
+    public Control secret = control(by.name("_.secret"));
```

Comment:
@amuniz just letting you know that I will be using something similar to what you've initially suggested (`control(by.name("_.secret"), by.name("secret"))`) but using the `path` instead of the name as suggested by Oliver. Thanks for the review!